base/cvd/cuttlefish/host/commands/cvd/server_command/serial_launch.cpp

```diff
@@ -264,11 +264,7 @@ class SerialLaunchCommand : public CvdServerHandler {
 
     size_t index = 0;
     for (const auto& device : devices) {
-      auto& mkdir_cmd = *req_protos.emplace_back().mutable_command_request();
-      *mkdir_cmd.mutable_env() = client_env;
-      mkdir_cmd.add_args("cvd");
-      mkdir_cmd.add_args("mkdir");
-      mkdir_cmd.add_args(device.home_dir);
+      CF_EXPECT(EnsureDirectoryExists(device.home_dir, 0775, ""));
```

Comment:
Not sure how relevant this is given it's probably safe to delete `serial_launch.cpp` entirely, but this is a behavioral change. It used to be deferred execution that `mkdir` was added to the sequence of commands, but now this happens immediately.

Reply:
I also think this should be deleted, will do it in the next PR.